package.json

```diff
@@ -45,8 +45,8 @@
   "homepage": "https://github.com/airbnb/hypernova-react#readme",
   "peerDependencies": {
     "hypernova": "^2.0.0",
-    "react": "0.14.x || >= 15.x",
-    "react-dom": "0.14.x || >= 15.x"
+    "react": "0.14.x || >= 15.x || >= 16.x",
```

Comment:
`>= 15.x` actually already includes 16.x, so these lines don't really need to change (altho they should probably change to `^0.14 || ^15 || ^16` instead of using `>=`

Reply:
done, however I'm not sure why this was even necessary from the start. Couldn't we just do >0.14? or do we have to ignore 0.15? In that case we should probably use ~0.14 instead

Reply:
updated to use ~0.14, I suspect we have to ignore 0.15 for some reason?

Reply:
There is no 0.15.